wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -132,15 +134,17 @@ public WireCompiler(String protoPath, List<String> sourceFileNames, List<String>
       throws WireException {
     this(new CommandLineOptions(protoPath, outputDirectory, sourceFileNames, roots,  registryClass,
         emitOptions,  new LinkedHashSet<String>(enumOptions), serviceWriterConstructor.getName(),
-        serviceWriterOptions));
+        serviceWriterOptions, false, false));
   }
 
   public WireCompiler(CommandLineOptions options) throws WireException {
-    this(options, new IO.FileIO());
+    this(options, new IO.FileIO(), new ConsoleWireLogger(options.quiet));
```

Comment:
Switching to dependency injection caused an issue here.  Berfore, the IO instance had a way to pull in an instance of the logger.  But now, if i wanted to do that, I'd have to instantiate the logger first, then pass it to the constructor of IO.FileIO and to the constructor for WireCompiler.  The this() redirection doesn't give you much opportunity to slip in extra code, so I'd have to do something with a static method.  Rather than complicating this code, I just dropped the log message out of IO.FileIO because it didn't seem important.